setup.cfg

```diff
@@ -74,3 +74,53 @@ universal = 1
 python-file-with-version = ldaptor/__init__.py
 history-file = docs/source/NEWS.rst
 tag-format = v{version}
+
+
+[flake8]
+disable-noqa = True
+max-line-length = 368
+extend-ignore =
+
+   E203,  # whitespace before : is not PEP8 compliant (& conflicts with black)
+
+   E101,  # indentation contains mixed spaces and tabs
```

Comment:
anything starting `Exxx` or `Wxxx` is a pycodestyle lint failure. `Fxxx` is for pyflakes

Reply:
almost all of these get fixed by black: https://github.com/twisted/ldaptor/pull/196/files#diff-380c6a8ebbbce17d55d50ef17d3cf906R85

Reply:
@adiroiban I'm going to do black in a follow-up PR as it's a huge number of changes and to review it I'm going to ask you to run "pipx run pre-commit run --all-files" until it passes and then get you to verify the diff